templates/user/signup.html

```diff
@@ -0,0 +1,22 @@
+{% extends "user/base.html" %}
+
+{% block contents %}
+<div>
+  <form role="form" action="{% url 'user:signup' %}" method="post" enctype="multipart/form-data">
```

Comment:
`multipart/form-data`もDjangoは対応していますが，デフォルトの`application/x-www-form-urlencoded`で問題ないです
```suggestion
  <form role="form" action="{% url 'user:signup' %}" method="post">
```

Reply:
追記: [参考](https://developer.mozilla.org/ja/docs/Web/HTML/Element/form#attr-enctype)